content/php/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'sqrt()'
+Description: 'Calculates the square root of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`sqrt()`** is a function that calculates the square root of a number. It takes a single paramter in the form of a number and returns the square root of that number.
+
+## Syntax
+
+```pseudo
+$result = sqrt($value)
+```
+
+- `$value` is the number whose square root is being calculated.
+- `$result` is the resultant square root value.
+
```

Comment:
You should note here that the result for a negative number is `NAN`

Reply:
I have added a note mentioning this case.